__tests__/KeyPairs.js

```diff
@@ -109,3 +90,6 @@ it('lets me open and close the keypair create modal', async () => {
   // Then the modal should be gone.
   expect(modal).not.toBeInTheDocument();
 });
+
+// it.only('lets me create a keypair', async () => {
+// });
```

Comment:
I think you forgot this here

Reply:
Ah yeah, this PR started as something else :)  thx